hack/specv2-discuss.yaml

```diff
@@ -0,0 +1,213 @@
+spec:
+  # Runtime
+  container:
+    command:
+      - python
+      - main.py
+    env:
+      - name: LOG_LEVEL
+        value: DEBUG
+      - name: PORT
+        value: "8080"
+    envFrom:
+      - configMap: my-cm
+      - secret: my-secret
+      - configMap: my-other-cm
+    filesFrom:
+      - secret:
+          name: my-other-secret
+          mountPath: /somewhere
+      - configMap:
+          name: my-third-cm
+          mountPath: /config
+      - emptyDir:
+          medium: Memory
+          mountPath: /tmp
+      - emptyDir:
+          medium: Disk
+          mountPath: /mnt
+      - persistentVolumeClaim:
+          name: my-pvc
+          mountPath: /tmp
+    image: ghcr.io/nais/example:20230302-cafebabe
+    liveness:
+      failureThreshold: 10
+      initialDelay: 20
+      path: /isalive
+      periodSeconds: 5
+      port: 8090
+      timeout: 1
+    readiness:
+      failureThreshold: 10
+      initialDelay: 20
+      path: /isalive
+      periodSeconds: 5
+      port: 8090
+      timeout: 1
+    startup:
+      failureThreshold: 10
+      initialDelay: 20
+      path: /isalive
+      periodSeconds: 5
+      port: 8090
+      timeout: 1
+    resources:
+      limits:
+        cpu: 500m
+        memory: 512Mi
+      requests:
+        cpu: 200m
+        memory: 256Mi
+  metrics:
+    enabled: true
+    path: /metrics
+    port: 8080
+  replicas:
+    min: 1
+    max: 5
+    autoscaling:
+      enabled: true
+      cpu: 50%
+      memory: 70%
+      kafka:
+        - topic: mytopic
+          group: losGroupos
+          maxLag: 123
+  leaderElection:
+    enabled: true
+  secureLogs:
+    enabled: false
+  caBundle:
```

Comment:
Syns denne er kjip med tanke på at den er veldig NAV-spesifikk. Samme gjelder `secureLogs`.  
I disse tilfellet ville f.eks Suffix vært fin, da man kunne laget den bare for NAV. Dok og alt vil da kun bli generert for NAV